liquibase-core/src/main/java/liquibase/logging/core/BufferedLogger.java

```diff
@@ -4,6 +4,7 @@
 
 import java.util.logging.Level;
 
+@Deprecated
 public class BufferedLogger extends AbstractLogger {
 
     private BufferedLogService bufferedLogService;
```

Comment:
We still use the `BufferedLogger` in the Pro CommandTests.  Should we just move that code to the Pro tests world?

Reply:
I was talking about this with @filipelautert, we also use it in [here](../blob/master/liquibase-cdi/src/test/java/liquibase/integration/cdi/CDILiquibaseTest.java#L31) in core. I created a [task](https://datical.atlassian.net/browse/DAT-14007) to figure out what to do with it.